### ISSUE TITLE
Fish hangs while trying to write a Git command after upgrading to 3.0

### ISSUE THREAD

**Issue by ktonga** [oh-my-fish/oh-my-fish#672]
To reproduce it you just have to start writing a git command and as soon as you type the first char after the git sub-command it will hang forever (eg: `git checkout m|`), I'm not even attempting to complete with tab so might be related to suggestions maybe or something else.

I let the terminal open for several minutes to see if it was just slow but it never comes back, I have to kill the terminal by closing the windows since Ctrl+C does not work either.

Git is the easiest way to reproduce this bug but it also happened to me with other commands such as `cd`. I use the theme `bobthefish` but it also happens with the default theme. If I disable OMF it does not happen, that's why I'm reporting it here.

Here is the output from `omf doctor`:

```
Oh My Fish version:   6
OS type:              Linux
Fish version:         fish, version 3.0.0
Git version:          git version 2.19.1
Git core.autocrlf:    no
Your shell is ready to swim.
```

Let me know if you need any more info or help debugging it.

Cheers.

**Comment by vonpupp** [comment]
I seem to be having the same issue here, herewith the `omf doctor`:
```
» omf doctor
Oh My Fish version:   6
OS type:              Linux
Fish version:         fish, version 3.0.0
Git version:          git version 2.20.1
Git core.autocrlf:    input
Your shell is ready to swim.
```

It seems to break only when I am on a git repository, either when I do a `git add <tab>` completion, or just when I just do an `ls`, sometimes even when I just enter a directory. I am sorry I cannot narrow it down. My only conclusion so far is it is related to omf + within a git repo.

**Comment by ktonga** [comment]
@vonpupp I use `fzf` but I don't have it installed via OMF.

This is my  `.config/omf/bundle`:

``` 
package cd
package fasd
package foreign-env
package https://github.com/jhillyerd/plugin-git
package local-config
theme bobthefish
theme default
```

**Comment by kavinvin** [comment]
I have this problem too. Mine seems coming from `fasd` in `omf`. The problem disappear after I remove it. And broke again after I reinstalled it.

Here's my plugin list
```
Plugins
fasd		fish-spec	git		omf

Themes
default		salmon-theme
```

**Comment by ktonga** [comment]
Can confirm. I have been using `omf` for a day or so without `fasd` plugin and the problem seems to be gone.

**Comment by ktonga** [comment]
Maybe related to https://github.com/oh-my-fish/plugin-fasd/issues/15

**Comment by pjrt** [comment]
Same here. Removed fasd and it works now.

**Comment by faho** [comment]
Hi!

I think this should be fixed via 4a3ac6e91ec415bfe536f722a71903d9a67bb10d in fish-shell/fish-shell (https://github.com/fish-shell/fish-shell/commit/4a3ac6e91ec415bfe536f722a71903d9a67bb10d), so the upcoming 3.0.1 release should fix it.

If anyone of you wants to test, build the Integration_3.0.1 branch (https://github.com/fish-shell/fish-shell/tree/Integration_3.0.1).

**Comment by ktonga** [comment]
Closing since it seems to be fixed in the recently released fish 3.0.1
